lib/xdrgen/generators/go.rb

```diff
@@ -165,14 +165,19 @@ def render_definition(out, defn)
         case defn
         when AST::Definitions::Struct ;
           render_struct out, defn
+          render_binary_interface out, name(defn)
         when AST::Definitions::Enum ;
           render_enum out, defn
+          render_binary_interface out, name(defn)
         when AST::Definitions::Union ;
           render_union out, defn
+          render_binary_interface out, name(defn)
         when AST::Definitions::Typedef ;
           render_typedef out, defn
+          render_binary_interface out, name(defn)
         when AST::Definitions::Const ;
           render_const out, defn
+          render_binary_interface out, name(defn)
```

Comment:
This probably won't compile for `const`.

Reply:
Good call, thanks. Fixed.